### DISCUSSION TITLE
FGDW002 Fibaro Door/Window Sensor 2 wakes up frequently every 5 minutes

### DISCUSSION THREAD

**Discussion by bub4** [zwave-js/zwave-js-ui#3605] [General]
Hi All, 

I switched from Home Assistant z-wave JS to *UI lately and I am blown away how good it is. 
Thank you guys so much!

I have a strange issue with Fibaros Door and Window Sensor 2 however.
Two of four sensors are working fine with a rather acceptable battery drain. 
The other two are draining battery like crazy. The sensors wake up every 5 minutes to communicate with the controller (zwave.me UZB). 
I've configured all 4 identcally. Those working not properly has been factory reset and re-interviewed many times. 
Setting the wakup interval parameter to the maximum doesn't help. It also doesn't matter if the sensor is next to the controller or far away - the sensor with the longest distance to the controller is working fine. 
All other Fibaro nodes are working fine (4 motion sensors, one button, one walli dimmer, one walli switch)

Fibaro obviously sais I should use their hardware... 

Is anyone here experienced with advanced troubleshooting? 

      "id": "40-132-0-wakeUpInterval",
      "nodeId": 40,
      "toUpdate": false,
      "commandClass": 132,
      "commandClassName": "Wake Up",
      "endpoint": 0,
      "property": "wakeUpInterval",
      "propertyName": "wakeUpInterval",
      "type": "number",
      "readable": false,
      "writeable": true,
      "label": "wakeUpInterval (property)",
      "default": 21600,
      "stateless": false,
      "commandClassVersion": 2,
      "min": 0,
      "max": 64800,
      "step": 3600,
      "list": false,
      "value": 64800,
      "lastUpdate": 1707938118856,
      "newValue": 64800

I would be glad if someone could give me a hint

**Comment by robertsLando** [comment]
> Fibaro obviously sais I should use their hardware...

LOL it's the device that reports the updates so that's the problem, you would have the same issues with ther hardware

Did you tried to update the firmware of your device? Maybe there was a know bug. Otherwise excluding, factory reset and re-including it could be a possible solution, if you configure each device in the same way it means that specific one has something wrong.

@AlCalzone any other suggestion?

Also please make a [driver log](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=driver-logs), loglevel `debug` and attach it here as a file (drag & drop into the text field).

**Comment by AlCalzone** [comment]
@bub4 are you sure it is waking up and not just reporting that frequently?
You already tried everything I'd suggest.

Maybe claim you're using SmartThings or something else that's Z-Wave certified, so they cannot blame the controller (which btw. has no influence on when the device wakes up aside from setting the interval).

**Reply by bub4** [reply]
@AlCalzone
I got one of their Yubii boxes and paired both of them. I can confirm that it is something related to the sensors.  They behaved just as crazy as with Zwave JS UI. One reported back every minute, the other every 5 minutes. I will try to get them replaced

**Comment by bub4** [comment]
Thank @robertsLando  @AlCalzone 
[zwave-js-ui-store.zip](https://github.com/zwave-js/zwave-js-ui/files/14293844/zwave-js-ui-store.zip)

Attached the debug log. Hope I did it right.
I made a factory reset and re-included it with standard values. After it woke up every 4-5 minutes I've adjusted the wakeup value to the maximum. The 4-5 minutes remained however.

I also believe that the sensors are sort of broken. Otherweise the other products wouldn't work properly. And they work flawlessly with z-wave JS UI. 

The Firmware is the same as the ones from the working sensors. Unfortunately they provide FW only via their boxes, btw. 

AlCalzone - I see the wakeup behaviour in the JS UI as well as in Home Assistant
<img width="414" alt="Bildschirmfoto 2024-02-15 um 10 05 16" src="https://github.com/zwave-js/zwave-js-ui/assets/77642206/74513eed-6ee2-4ba2-a96f-c5865586cab3">
